l1-contracts-foundry/script-config/deploy-l1.toml

```diff
@@ -1,6 +1,6 @@
 deployer_private_key = "0x0000000000000000000000000000000000000000000000000000000000000000"
 gas_price = 0
-chain_id = 9
+chain_id = 31337
```

Comment:
actually you can switch to main branch of zksync-era, and there it's reth with chain_id 9

Reply:
Removed, now it's picked up from `block.chainid`.